docs/source/sysadmin_guide/configuring_sawtooth/rest_api_configuration_file.rst

```diff
@@ -0,0 +1,64 @@
+---------------------------
+REST API Configuration File
+---------------------------
+
+The REST API configuration file specifies network connection settings and an
+optional timeout value.
+
+If the config directory contains a file named ``rest_api.toml``, the
+configuration settings are applied when the REST API starts.  Specifying a
+command-line option will override the setting in the configuration file.
+
+Note: By default, the config directory is /etc/sawtooth/.
```

Comment:
Consider making this an RST note. Also, /etc/sawtooth/ should have command formatting to be consistent